ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/ApiTestClientBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using RestSharp;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public class ApiTestClientBase : IApiClient
```

Comment:
Looks like you created a Base class just for the reason of having the Get, Post, Put, Delete methods to return the RestRequest, I would say just have private methods in the BookingClient.